packages/cli/src/commands/plugin/build.ts

```diff
@@ -14,43 +14,17 @@
  * limitations under the License.
  */
 
-import { Command } from 'commander';
-import fs from 'fs-extra';
-import path from 'path';
-import recursive from 'recursive-readdir';
-import { run } from '../../helpers/run';
+const rollup = require('rollup'); // "import" is not working for some reason...
+import rollupConfig from './rollup.config';
 
-const copyStaticAssets = async () => {
-  const pluginRoot = fs.realpathSync(process.cwd());
-  const source = path.resolve(pluginRoot, 'src');
-  const destination = path.resolve(pluginRoot, 'dist', 'cjs');
-  const assetFiles = await recursive(source, [
-    '**/*.tsx',
-    '**/*.ts',
-    '**/*.js',
-  ]);
-  assetFiles.forEach(file => {
-    const fileToBeCopied = file.replace(source, destination);
-    const dirForFileToBeCopied = path.dirname(fileToBeCopied);
-    fs.ensureDirSync(dirForFileToBeCopied);
-    fs.copyFileSync(file, file.replace(source, destination).toString());
-  });
-};
-
-export default async (cmd: Command) => {
-  const args = [
-    '--outDir',
-    'dist/cjs',
-    '--noEmit',
-    'false',
-    '--module',
-    'CommonJS',
-  ];
-
-  if (cmd.watch) {
-    args.push('--watch');
-  }
+export default async () => {
+  const inputOptions = {
+    input: rollupConfig.input,
+    plugins: rollupConfig.plugins,
+  };
+  const outputOptions = rollupConfig.output;
 
-  await copyStaticAssets();
-  await run('tsc', args);
+  const bundle = await rollup.rollup(inputOptions);
+  await bundle.generate(outputOptions);
+  await bundle.write(outputOptions);
```

Comment:
This doesn't support watch mode right? We need support for that for ´yarn start´ to work